dnat-emulators

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+DEF_EMULATOR_IP=172.17.0.1
+
+EMULATOR_IP=$(getent hosts host.docker.internal|awk '{ print $1 }')
+
+if [[ ! $EMULATOR_IP ]]; then
+    EMULATOR_IP=$DEF_EMULATOR_IP
+fi
+
+for ((i=0; i<10; i++)); do iptables -t nat -A OUTPUT -p tcp -d 10.220.220.$((1+i)) --dport 22 -j DNAT --to-destination $EMULATOR_IP:$((2223+i)); done
```

Comment:
nice, I would do some polishing here:

- add "This file is part of Sailfish SDK" headline
- wrap the "for" line
- space before '|' :)
- ideally store out of PATH, maybe under /usr/libexec/sdk-setup
- move the FIXME from .service to here

Reply:
Fixed all these. Ended up adding whole BSD header, as the license is not clear if someone stumbles upon the file on the build engine. Feels a bit funny to add a long copyright header to a few lines of code, but there it is now :)